portfolio/src/index.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+html {
```

Comment:
I know its usually not the case that you would ever want to use CSS when incorporating Tailwind into your project, but this is a rare exception to the rule. I actually was able to add a cool scrolling effect to the site for whenever the user clicks on one of the header menu options to be navigated to a specific section on the page. With this addition, the page will actually quickly animate the scroll of the page down to the specified section, which I thought would be a cool addition to make, especially since it only requires 3 lines of code.